scripts/acceptance.sh

```diff
@@ -6,5 +6,4 @@ mocha \
   --retries 5 \
   --require test.config.js \
   -t 60000 \
-  $@ \
-  dotenv_config_path=.env.test
```

Comment:
Looks like we don't need this, given we specified this [manually in test.config.js](https://github.com/artsy/force/blob/e86d4a39d2ae2769c483eb9cce9aafddb8ca35a1/test.config.js#L21). In fact, on CircleCI it's currently raising a warning treating this as a test file:

<img width="1288" alt="screen shot 2018-04-07 at 10 33 20 pm" src="https://user-images.githubusercontent.com/796573/38462350-ddbffbca-3ab3-11e8-95a3-5926af765659.png">